test/test_sparse.py

```diff
@@ -952,6 +952,10 @@ def test_permute(self, device, dtype, coalesced, gradcheck):
             s.permute(dims=(1, 0))
         with self.assertRaisesRegex(RuntimeError, "duplicate dims"):
             s.permute(dims=(1, 1, 1))
+        # This edge case used to segfault
```

Comment:
Nit: if you are going to put a comment in here, at least add an issue number to make it more clear what we are testing.